src/test/java/com/nephest/battlenet/sc2/discord/GuildEmojiStoreTest.java

```diff
@@ -0,0 +1,99 @@
+package com.nephest.battlenet.sc2.discord;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import com.nephest.battlenet.sc2.model.Race;
+import com.nephest.battlenet.sc2.model.BaseLeague.LeagueType;
+import discord4j.common.util.Snowflake;
+import discord4j.core.event.domain.interaction.InteractionCreateEvent;
+import discord4j.core.object.command.Interaction;
+import discord4j.core.object.entity.Guild;
+import discord4j.core.object.entity.GuildEmoji;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@ExtendWith(MockitoExtension.class)
+public class GuildEmojiStoreTest {
+    
+    @Mock
+    private InteractionCreateEvent evt;
+ 
+    @Mock
+    private Interaction interaction;
+
+    @Mock
+    private Guild guild;
+
+    @Mock
+    private GuildEmoji diamondEmoji;
+
+    @Mock
+    private GuildEmoji terranEmoji;
+
+    @Mock
+    private GuildEmoji otherEmoji;
+
+    private GuildEmojiStore guildEmojiStore;
+
+    @BeforeEach
+    public void beforeEach() {
+        System.out.println("beforeEach");
```

Comment:
This probably are debugging leftovers and should be removed, or this serves some purpose?

Reply:
This is, as you say, just some debugging leftover when I was mixing test annotations without realising and scratching my head why the before each wasn't being executed. I will remove.